catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/consumes.xml

```diff
@@ -0,0 +1,183 @@
+                    availability="optional"/>
+    <reference id="webBranding" interface="org.codice.ddf.branding.BrandingPlugin"/>
+    <reference-list id="historianConfigurations"
+                    interface="ddf.catalog.configuration.HistorianConfiguration"
+                    availability="optional">
+        <reference-listener bind-method="bind"
+                            unbind-method="unbind" ref="configurationApplication"/>
+    </reference-list>
+    <reference id="resourceBundleLocator"
+               interface="ddf.platform.resource.bundle.locator.ResourceBundleLocator"/>
+
+    <!--
+    =============================================================
+    SIMPLE APPS
+    (AUTH/BUILD/CATALOG/PLATFORM/FEEDBACK/GEOCODER/LOGOUT/SESSION/USER)
+    =============================================================
+    -->
+
+    <reference id="securityAuthenticationService"
+               interface="org.codice.ddf.security.rest.authentication.service.AuthenticationService"/>
+
+    <reference-list id="userCreatableMetacardType"
+                    interface="org.codice.ddf.catalog.ui.metacard.internal.UserCreatableMetacardType"
+                    availability="optional"/>
+
+    <reference id="platformConfigurationService"
+               interface="org.codice.ddf.configuration.service.PlatformUiConfigurationService"/>
+
+    <reference id="geoCoderService" interface="org.codice.ddf.spatial.geocoding.GeoCoderService"/>
+
+    <reference id="logoutService" interface="org.codice.ddf.security.logout.service.LogoutService"/>
+
+    <reference id="sessionManagementService"
+               interface="org.codice.ddf.security.session.management.service.SessionManagementService"/>
+
+    <!--
+    =============================================================
+    DESCRIPTORS APPLICATION
+    =============================================================
+    -->
+
+    <bean id="metacardTransformers" class="ddf.catalog.util.impl.SortedServiceReferenceList"/>
+    <reference-list interface="ddf.catalog.transform.MetacardTransformer"
+                    member-type="service-reference">
+        <reference-listener ref="metacardTransformers" bind-method="bindService"
+                            unbind-method="unbindService"/>
+    </reference-list>
+
+    <!--
+    =============================================================
+    LIST APPLICATION
+    =============================================================
+    -->
+
+    <reference id="mimeTypeMapper" interface="ddf.mime.MimeTypeMapper"/>
+
+    <!--
+    =============================================================
+    METACARD APPLICATION
+    =============================================================
+    -->
+
+    <reference-list id="reportingMetacardValidators"
+                    interface="ddf.catalog.validation.ReportingMetacardValidator"
+                    availability="optional"/>
+
+    <!-- TODO - Shouldn't there only be one registry? -->
```

Comment:
✏️  It might be worth fixing the discrepancy with how we inject the registry. It's done inconsistently; both by injecting a singleton reference, and also injecting a list and grabbing the first one. I don't see a reason for this.

Reply:
Moving both to the `consumes.xml` file and leaving a note that impl details impact this.